src/test/resources/jobs_response.json

```diff
@@ -0,0 +1,24 @@
+{
+  "return": [
+    {
+      "20150304192951636258": {
+        "Arguments": [],
+        "Function": "test.ping",
+        "StartTime": "2015, Mar 04 19:29:51.636258",
+        "Target": "*",
+        "Target-type": "glob",
+        "User": "chuck"
+      },
+      "20150304200110485012": {
+        "Arguments": [
+          "enable-autodestruction"
```

Comment:
Autodestruction initiated by Chuck is lethal in most cases, but I was lucky and escaped it :D